src/main/resources/plugin.yml

```diff
@@ -140,6 +140,8 @@ permissions:
     description: Can set his clan's banner
   simpleclans.leader.discord.create:
     description: Can create discord channel for his clan
+  simpleclans.leader.rename:

```

Comment:
`false` by default

Reply:
Why `false`? Leaders shouldn't have the possibility to rename their clan by default?

Reply:
Yeah, some servers might not want that.